src/main/java/org/jabref/gui/util/DummyFileUpdateMonitor.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.gui.util;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import org.jabref.model.util.FileUpdateListener;
+import org.jabref.model.util.FileUpdateMonitor;
+
+/**
+ * This {@link FileUpdateMonitor} does nothing.
+ * Normally, you want to use {@link DefaultFileUpdateMonitor} except if you don't care about updates.
+ */
+public class DummyFileUpdateMonitor implements FileUpdateMonitor {
```

Comment:
Since the `FileUpdateMonitor` interface is in model, why don't you just put this class into the exact same package? It would solve the dependency problems that are still breaking the build.

Reply:
Good observation! I moved the file and so fixed the build.